include/dem/boundary_cells_info_struct.h

```diff
@@ -1,6 +1,6 @@
 /* ---------------------------------------------------------------------
  *
- * Copyright (C) 2019 - 2019 by the Lethe authors
+ * Copyright (C) 2019 - 2024 by the Lethe authors
```

Comment:
There are some files in Lethe that do not have a second date, and are like these:
`Copyright (C) 2019 - by the Lethe authors`
What is the right approach for this? Let's decide on something so we can have the same across all files.

Reply:
I agree

Reply:
I see deal.II uses the same format (year change with the update of the year). 
But I think this is not mandatory, and only the publication date is necessary (legally) https://liferay.dev/blogs/-/blogs/how-and-why-to-properly-write-copyright-statements-in-your-code#why-not-bump-the-year-on-change.
What do you think?

Reply:
In that case I guess we can do it in the same way as deal.II